botfront/imports/ui/components/stories/Stories.jsx

```diff
@@ -137,6 +140,11 @@ function Stories(props) {
 
     const handleGetResponseLocations = (responses, f) => { Meteor.call('stories.includesResponse', projectId, responses, wrapMeteorCallback(f)); };
 
+    useEventListener('keydown', ({ key }) => {
+        if (['INPUT', 'TEXTAREA'].includes(document.activeElement.tagName)) return;
+        if (key === 'ArrowLeft') treeRef.current.focusMenu();
+    });
```

Comment:
could you explain what this is supposed to do? I couldn't figure it out

Reply:
Yeah, it's a half baked feature unrelated with activity: it allows a user that's been editing stories to hit the left arrow to refocus the story menu, and thus be able once again to use the up down arrows to navigate.